docs/howto/openstack/cinder/encrypted-volumes.md

```diff
@@ -97,57 +89,33 @@ openstack volume show \
 
-However, if you are trying to delete a volume that was created by a
-different user, you’ll run into the limitation that the *secret*
-associated with the volume is owned by that user. As a result, the
-deletion of the encrypted volume using your own user credentials will
-fail.
+However, if you are trying to delete a volume that was created by a different user, you’ll run into the limitation that the *secret* associated with the volume is owned by that user.
+As a result, the deletion of the encrypted volume using your own user credentials will fail.
 
 There are two options to work around this limitation:
 
-1. You can switch to the user credentials of the user that created the
-   volume (if you have access to them), and proceed with the deletion.
-2. You can ask the user that created the volume to [add you to the
-   Access Control List (ACL) for the
-   secret](../barbican/share-secret.md). This will enable you to read
-   the secret, and to delete the volume using your own credentials.
+1. You can switch to the user credentials of the user that created the volume (if you have access to them), and proceed with the deletion.
+2. You can ask the user that created the volume to [add you to the Access Control List (ACL) for the secret](../barbican/share-secret.md).
+   This will enable you to read the secret, and to delete the volume using your own credentials.
 
 
 ## Block device encryption caveats
 
-Once a volume is configured to use encryption and is also attached to
-an instance in {{brand}}, some caveats apply that you might want
-to keep in mind.
+Once a volume is configured to use encryption and is also attached to an instance in {{brand}}, some caveats apply that you might want to keep in mind.
 
-Sometimes, automatically or through administrator intervention, we
-move one of your instances to another physical machine. This process
-is known as *live migration,* and it normally does not interrupt the
-instance’s functionality at all — typically, neither you nor the
-application users notice that live migration has even happened. This
-is a very common occurrence when we do routine upgrades of the
-{{brand}} platform, during our pre-announced maintenance windows.
+Sometimes, automatically or through administrator intervention, we move one of your instances to another physical machine.
+This process is known as *live migration,* and it normally does not interrupt the instance’s functionality at all — typically, neither you nor the application users notice that live migration has even happened.
+This is a very common occurrence when we do routine upgrades of the {{brand}} platform, during our pre-announced maintenance windows.
 
-The same considerations apply to physical node failure. If the
-physical machine running your instance fails, we can automatically
-recover it onto another machine — an action known as *evacuation.*
+The same considerations apply to physical node failure.
```

Comment:
Since we're talking in general, the plural form sounds better to me:

```suggestion
The same considerations apply to physical node failures.
```

Reply:
I disagree, and prefer the singular as it was.